tools/mkpkg/update_retroplayer-addons

```diff
@@ -111,15 +104,16 @@ for addontxt in "binary-addons https://github.com/lrusak/repo-binary-addons.git
 
           OLD_HASH=$(get_pkg_var "${EMULATOR}" PKG_VERSION)
           PKG_SITE=$(get_pkg_var "${EMULATOR}" PKG_SITE)
-          git_clone $PKG_SITE master $EMULATOR.git
+          GIT_HASH=$(git ls-remote $PKG_SITE master | awk '{print $1}')
```

Comment:
it updates from master branch also if a other branch is defaulted at the repo (libretro-fsuae for example)

might need to change it to `HEAD` instead of `master`